tests/packaging/images/Dockerfile.debian-9-stretch

```diff
@@ -1,7 +1,7 @@
 FROM debian:stretch-slim
 
 RUN apt-get update &&\
-    apt-get install -yq ca-certificates procps systemd wget libcap2-bin apt-transport-https
+    apt-get install -yq ca-certificates procps systemd wget libcap2-bin apt-transport-https systemd-sysv
```

Comment:
What is this for? We shouldn't really be using sysv on Debian 9 since it already has systemd.

Reply:
I can revert that... I added that package to install /sbin/shutdown when I was testing running "shutdown" in the container for the system restart tests as a workaround for the docker restart issues.